src/WorkOS.net/Services/UserManagement/_interfaces/AuthenticateUserWithMagicAuthOptions.cs

```diff
@@ -32,10 +32,10 @@ public class AuthenticateUserWithMagicAuthOptions : BaseOptions
         public string Code { get; set; }
 
         /// <summary>
-        /// The challenge ID returned from when the one-time code was sent to the user.
+        /// The Id of the User authenticating.
```

Comment:
In comments we'll want to use "ID" instead of "Id":

```suggestion
        /// The ID of the User authenticating.
```